_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+"Back In Black".
+
+<iframe src="https://www.youtube.com/embed/c_qUp0SGeKE"  
+width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" 
+scrolling="no" style="border:1px solid #CCC; border-width:1px; 
+margin-bottom:5px; max-width: 100%;" allowfullscreen> 
+</iframe>
+
+## 2022 Roadmap: Project Tardigrade
+
+One of the interesting developments that came out of Trino Summit was a feature
+Trino co-creator, Martin, talked about in [the State of Trino presentation](https://www.starburst.io/resources/trino-summit/?wchannelid=2ug6mgs5ao&wmediaid=o264qw85dj).
+He discussed a prioritization in granular fault tolerance and performance in the
+core engine. While Trino has been proven at scale to run batch processes, many
+have avoided this for long-running batch jobs in fear of a failure during longer
+running batch jobs. The fault-tolerance feature introduces a direction that the
+Trino project is taking that will ease the minds of its users to adopt it as a
+long-running batch query engine.
+
+The granular fault-tolerance is being thoughtfully crafted to maintain the 
+speed that Trino has over other query engines in this space, while increasing
+the resiliency of queries. In other words, rather than when a query runs out of
+resources or fails for any other reason, a subset of the query (the task) will
+be retried. To support this intermediate stage data will be persisted to 
+replicated RAM or SSD.
+
+<a title="Schokraie E, Warnken U, Hotz-Wagenblatt A, Grohme MA, Hengherr S, et al. (2012), CC BY 2.5 &lt;https://creativecommons.org/licenses/by/2.5&gt;, via Wikimedia Commons" 
+ href="https://commons.wikimedia.org/wiki/File:SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png"><img width="512" 
+ alt="SEM image of Milnesium tardigradum in active state - journal.pone.0045682.g001-2" src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png/512px-SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png">
+</a>
+
+We're happy to announce that all these efforts to introduce granular
+fault-tolerance into Trino we are calling Project Tardigrade. This project is
+named after the microscopic Tardigrades that are the worlds most indestructible
+creatures. We look forward to telling you more about this project in the coming
+year.
+
+## Conclusion
+
+In summary, living this first year under the banner of Trino was nothing short
+of a wild endeavor. Any engineer knows that naming things is hard, and renaming
+things is all the more difficult. 
+
+As we head into 2022, we can be certain of one thing. Trino will be reaching 
+into newer areas of development and breaking norms just as it did as Presto in 
+previous eras. The adoption of native fualt-tolerance to a lightning fast query
+engine will bring Trino to a new level of adoption. Keep your eyes peeled for 
+more about Project Tardigrade.
```

Comment:
needs a final sentence that talks about the fact that lots of other features will come and will be contributed by all the great members of the community